microsetta_admin/server.py

```diff
@@ -262,6 +262,9 @@ def email_stats():
     if project == "":
         project = None
 
+    # de-duplicate
+    emails = list(set(emails))
```

Comment:
loses deterministic behavior- if we care, maybe sorted(list(set(emails)))?  I thought about doing this and leaned towards output indexing matching input indexing perfectly to simplify programmatic parsing, but if all we care about is returning a new xml file it should be fine.

Reply:
no requirement to index match the uploaded input